### PR TITLE
PR: Print last line of Pdb code

### PR DESCRIPTION
**Pull request by impact27**
## Description of Changes

- When entering code in IPython, if the last line is an expression, it is printed after an "Out [i]: " prompt. This PR reproduces this behaviour in the debugger.
- Depends on spyder-ide/spyder-kernels#355

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

### CODE REVIEW

File: spyder/plugins/ipythonconsole/widgets/debugging.py [hunk 1]
```diff
@@ -431,6 +431,20 @@ def set_pdb_state(self, pdb_state):
         if pdb_state is not None and isinstance(pdb_state, dict):
             self.refresh_from_pdb(pdb_state)
 
+    def pdb_out(self, text):
+        """Show pdb out."""
```
Comment (ccordoba12) [inline spyder/plugins/ipythonconsole/widgets/debugging.py:435]:
```suggestion
    def show_pdb_output(self, text):
        """Show Pdb output."""
```

Let's rename this method to make clearer what it does.

File: spyder/plugins/ipythonconsole/widgets/shell.py [hunk 2]
```diff
@@ -131,6 +131,7 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         handlers.update({
             'pdb_state': self.set_pdb_state,
             'pdb_execute': self.pdb_execute,
+            'pdb_out': self.pdb_out,
```
Comment (ccordoba12) [inline spyder/plugins/ipythonconsole/widgets/shell.py:134]:
```suggestion
            'show_pdb_output': self.show_pdb_output,
```

**Review by ccordoba12 — COMMENTED** [review]
Thanks @impact27! I left some minor suggestions for you, otherwise looks good to me.

**Comment by impact27** [comment]
@ccordoba12 I added a return value to runcell and runfile following the same logic

**Comment by ccordoba12** [comment]
> I added a return value to runcell

Nice! I think that's a great addition!

> and runfile following the same logic

I'm not sure about that. I mean, `runfile` is our equivalent of `python myfile.py`, so I think we should preserve that you need to print something if you want to observe an output in that case.

**Comment by impact27** [comment]
@ccordoba12 removed for runfile

**Pull request merged**